src/main/java/seedu/address/ui/UserProfileWindow.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.person.User;
+
+public class UserProfileWindow extends UiPart<Stage> {
```

Comment:
Should probably have a javadocs here. Don't need so fancy. Just a simple description will do.